docs/runtimes/python-based-custom-runtime-example/Python-Based-Custom-Runtime-with-Model-Stored-on-PVC.md

```diff
@@ -0,0 +1,403 @@
+# The Python-Based Custom Runtime with Model Stored on Persistent Volume Claim
+
+This document can provide the step by step instructions to demonstrate how to write a custom python-based `ServingRuntime` of `MLModel` and store the trained models on persistent volume claims.
+
+Before writing the custom `ServingRuntime`, user should have a K8s cluster with `kserve modelmesh` installed firstly by following [doc](https://github.com/kserve/modelmesh-serving/blob/main/docs/quickstart.md). Please make sure `modelMesh serving` is working well, it looks like below:
+
+The `modelMesh serving` pods are running:
+
+```shell
+kubectl get pods
+
+NAME                                    READY   STATUS    RESTARTS   AGE
+etcd-78ff7867d5-h4r2t                   1/1     Running   0          5m
+minio-6b85d9b464-nmk5b                  1/1     Running   0          5m
+modelmesh-controller-695b8488f5-hndgn   1/1     Running   0          5m
+```
+
+The `ServingRuntime`s are available: 
+
+```shell
+kubectl get servingruntimes
+
+NAME             DISABLED   MODELTYPE     CONTAINERS   AGE
+mlserver-1.x                sklearn       mlserver     2m
+ovms-1.x                    openvino_ir   ovms         2m
+torchserve-0.x              pytorch-mar   torchserve   2m
+triton-2.x                  keras         triton       2m
+```
+
+# Deploy a model stored on a Persistent Volume Claim
+
+Let's use namespace `modelmesh-serving` here:
+
+```shell
+kubectl config set-context --current --namespace=modelmesh-serving
+```
+
+## 1. Create PV and PVC for storing model file
+
+```shell
+kubectl apply -f - <<EOF
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: my-models-pv
```

Comment:
```suggestion
  name: my-models-pvc
```

Reply:
Hi @rafvasq, this yaml file firstly create persistent volume, then persistent volume claim, so I use the different name for it.